### PR TITLE
denose some of the remaining apps - abuse to discovery

### PR DESCRIPTION
**Pull request by eviljeff**

### CODE REVIEW

File: src/olympia/bandwagon/tests/test_views.py [hunk 1]
```diff
@@ -293,9 +292,9 @@ def test_publisher(self):
         CollectionUser.objects.create(collection=self.c, user=u)
         self.client.login(username='fligtar@gmail.com', password='foo')
         r = self.client.get(self.url)
-        eq_(r.status_code, 200)
+        assert r.status_code == 200
         # TODO(cvan): Uncomment when bug 719512 gets fixed.
-        #eq_(pq(r.content)('.meta .view-stats').length, 1,
+        # assert pq(r.content)('.meta .view-stats').length == 1, (
```
Comment (eviljeff) [inline src/olympia/bandwagon/tests/test_views.py]: Leaving this as TODO makes me sad as bug 719512 was fixed years ago but I don't know enough about the collections pages to fix (.meta .view-stats is always empty)

File: src/olympia/addons/tests/test_views.py [hunk 2]
```diff
@@ -359,17 +359,17 @@ class TestDeveloperPages(TestCase):
     def test_meet_the_dev_title(self):
         r = self.client.get(reverse('addons.meet', args=['a592']))
         title = pq(r.content)('title').text()
-        assert title.startswith('Meet the Gmail S/MIME Developer') == True
```
Comment (EnTeQuAk) [inline src/olympia/addons/tests/test_views.py]: Ugh, nice find :fearful:

File: src/olympia/bandwagon/tests/test_views.py (hunk 1 above)
Comment (EnTeQuAk) [inline src/olympia/bandwagon/tests/test_views.py]: Would be out of the scope of the PR anyway imho so don't worry :)

File: src/olympia/editors/tests/test_sql_model.py [hunk 3]
```diff
@@ -136,13 +135,13 @@ def test_get(self):
         c = Summary.objects.all().having('total =', 1).get()
         assert c.category == 'apparel'
 
-    @raises(Summary.DoesNotExist)
     def test_get_no_object(self):
-        Summary.objects.all().having('total =', 999).get()
+        with self.assertRaises(Summary.DoesNotExist):
+            Summary.objects.all().having('total =', 999).get()
```
Comment (EnTeQuAk) [inline src/olympia/editors/tests/test_sql_model.py]: Or use `pytest.raises` here too but I guess it doesn't really matter.

**Comment by EnTeQuAk** [comment]
r+wc once travis is happy, otherwise only minor nitpicks.

**Pull request merged**